features/step_definitions/basic_steps.rb

```diff
@@ -1,3 +1,20 @@
 Then("I should see {string}") do |content|
   expect(page).to have_content content
 end
+
+Then("{string} is present") do |selector|
+  expect(page).to have_selector :css, "#navigationBar"
+  expect(page).to have_selector :css, "#footer"
+end
```

Comment:
Can't see that you are making use of `selector`

Reply:
ask @diraulo :)

Reply:
About what? Is this @diraulo 's code or yours`